pystac/validation/stac_validator.py

```diff
@@ -159,9 +159,34 @@ def get_schema_from_uri(self, schema_uri: str) -> Tuple[Dict[str, Any], Any]:
 
         return schema, resolver
 
-    def _validate_from_uri(self, stac_dict: Dict[str, Any], schema_uri: str) -> None:
-        schema, resolver = self.get_schema_from_uri(schema_uri)
-        jsonschema.validate(instance=stac_dict, schema=schema, resolver=resolver)
+    def _validate_from_uri(
+        self,
+        stac_dict: Dict[str, Any],
+        stac_object_type: STACObjectType,
+        schema_uri: str,
+        extension_id: Optional[str] = None,
+        href: Optional[str] = None,
+    ) -> None:
+        try:
+            schema, resolver = self.get_schema_from_uri(schema_uri)
```

Comment:
Can you add a comment explaining where this code comes from? (cribbed from the **jsonschema** source)

Reply:
attrib'ed